src/update/UpdateDataParser.cpp

```diff
@@ -56,23 +56,41 @@ bool UpdateDataParser::HandleXMLTag(const wxChar* tag, const wxChar** attrs)
         return true;
     }
 
-    if (wxStrcmp(tag, mXmlTagNames[XmlParsedTags::kWindowsTag]) == 0)
+    const wxPlatformInfo& info = wxPlatformInfo::Get();
+
+    constexpr bool is32Bit = sizeof(void*) == 4;
+    constexpr bool is64Bit = sizeof(void*) == 8;
```

Comment:
This may be no problem, but is `#ifdef _WIN64` the more proper documented way to test it?

Reply:
Actually, both are wrong in this case :) They will match ARM64 as well.

But _WIN64 is not supported in Cygwin or Mingw and we have an ongoing task to unblock the support